deu/match-cleared.vbs

```diff
@@ -36,16 +36,16 @@ END IF
 
 'END FUNCTIONS LIBRARY BLOCK================================================================================================
 script_run_lowdown = ""
-'TODO I need error proofing in multiple places on this script. in and out of IULA and IULB ensuring the case and on CCOL' 'need to check about adding for multiple claims'
```

Comment:
There was a lot of code in this update that was out of the scope of the issue. I did not test this or heavily review it.

Reply:
removed

Reply:
it looks like a lot but it was just spaces and i am not sure why it came up the way it did